tasks/start.js

```diff
@@ -22,25 +26,43 @@ var runBuild = function () {
     });
 
     build.on('close', function (code) {
```

Comment:
it doesn't have to be, it'll call any function with the same arguments that you get in the anonymous function